### DISCUSSION TITLE
CardDAV Settings for Radicale

### DISCUSSION THREAD

**Discussion by C00kiekiller** [blind-coder/rcmcarddav#475] [Q&A]
Hi,

I use Radicale 3.0.6 with IMAP Login on a Ubuntu 22.04 Server.
I want to sync my contacs in Roundcube Webmail 1.6.7 with CardDAV v5.1.0.

But i don't know how. What are the needed values for the Discovery URL?

DAVx5 on Android works like a charm.

Username is: foo@bar.foo
Domain is: http://foo.bar:5232

Radicale Web Gui shows fot the addressbook url: 
https://foo.bar:5232/foo%40bar.foo/17c68e3b-486c-f524-f2c0-f94f86617ffa/

```
pip3 show radicale-imap
Name: radicale-imap
Version: 0.2.0
Summary: Radicale IMAP authentication plugin
Home-page: https://gitlab.com/comzeradd/radicale-imap/
Author: Nikos Roussos
Author-email: nikos@roussos.cc
License: LICENSE
Location: /usr/local/lib/python3.10/dist-packages
Requires: radicale
Required-by:

```

Regards

**Comment by mstilkerich** [comment]
Hi, the URL of the addressbook should work, even https://foo.bar:5232/ should work.

Since you are using https, it might be an issue with a certificate that is not signed by an authority trusted by the server running roundcube.

If you want to see whats the problem, turn on debug logging and take a log at carddav_http.log. If you want be to take a look, please provide the log file.

I have described here how to enable debug logging: https://github.com/mstilkerich/rcmcarddav/issues/474#issuecomment-2252954588

**Reply by C00kiekiller** [reply]
Hi, thanks for the hint with the Debug logging. This gaves me the right direction to solve my issue.
TSL Handshake failed, because the Let's Encrypt Root CA certificates are missing.
Added the missing certificates, now it's working like a charm.

Thx

Regards :-)

**Comment by C00kiekiller** [comment]
Problem solved after adding the missing Let's Encrypt certificates.
